geocoding/lib/geocoding.dart

```diff
@@ -15,9 +15,9 @@ export 'package:geocoding_platform_interface/geocoding_platform_interface.dart';
 /// When not supplied the currently active locale of the device will be used.
 /// The `localeIdentifier` should be formatted using the syntax:
 /// [languageCode]_[countryCode] (eg. en_US or nl_NL).
-Future<List<Location>> locationFromAddress(
+Future<List<Location>?> locationFromAddress(
```

Comment:
I wonder if it were better to return an empty list than a nullable list? I can then use first or default as a consumer of the library rather than checking if the list itself is null.